lms/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -0,0 +1,41 @@
+"""
+Classes that override default django-oauth-toolkit behavior
+"""
+
+from django.contrib.auth import authenticate, get_user_model
+from oauth2_provider.oauth2_validators import OAuth2Validator
+
+
+class EdxOAuth2Validator(OAuth2Validator):
+    """
+    Validator class that mimics the default behavior, but does not require
+    users to be validated.
```

Comment:
Nit: "validated" -> "active"
Also, did you want to mention allowing identification by email address in addition to username?